icu4j/perf-tests/src/main/java/com/ibm/icu/text/BreakTransliteratorAccess.java

```diff
@@ -2,8 +2,38 @@
 // License & terms of use: http://www.unicode.org/copyright.html
 package com.ibm.icu.text;
 
+/**
+ * This provides access for testing to the non-public {@code BreakTransliterator}.
+ *
+ * @internal
+ * @deprecated This API is ICU internal only.
+ */
+@Deprecated
```

Comment:
Why do we need these docs tags in perf-tests? Why are API tools looking at perf-tests?

Reply:
I guess worst case we could invent a doc tag for a class that tells the API code to ignore it even if it's public and in what looks like an API package. It should be possible to tag the class and not have to tag every public member of it.

Reply:
Will see what the right fix should be.

This is new. We didn't have classes in the `com.ibm.icu.text` package before (outside main/)
So I guess this is something nobody imagined.